backend/src/main/kotlin/com/tngtech/apicenter/backend/connector/database/entity/AccessRecordEntity.kt

```diff
@@ -0,0 +1,28 @@
+package com.tngtech.apicenter.backend.connector.database.entity
+
+import org.hibernate.search.annotations.IndexedEmbedded
+import javax.persistence.*
+
+@Entity
+class AccessRecordEntity(
+        @EmbeddedId @IndexedEmbedded val accessRecordId: AccessRecordId,
+
+        @ManyToOne
```

Comment:
As far as I understand, yes, because the parent (ServiceEntity) needs a reference so the remove cascade can remove AccessRecordEntities when the ServiceEntity is deleted. The child needs a reference because the ServiceId is part of it's primary key.

Reply:
Okay, that (unfortunately) makes sense. Thanks!